resources/functionMap.php

```diff
@@ -9467,222 +9467,269 @@
 'RecursiveTreeIterator::setPostfix' => ['void', 'prefix'=>'string'],
 'RecursiveTreeIterator::setPrefixPart' => ['void', 'part'=>'int', 'prefix'=>'string'],
 'RecursiveTreeIterator::valid' => ['bool'],
-'Redis::__construct' => ['void'],
-'Redis::_prefix' => ['string', 'value'=>'mixed'],
-'Redis::_serialize' => ['mixed', 'value'=>'mixed'],
-'Redis::_unserialize' => ['mixed', 'value'=>'string'],
-'Redis::append' => ['int', 'key'=>'string', 'value'=>'string'],
-'Redis::auth' => ['bool', 'password'=>'string|string[]'],
-'Redis::bgRewriteAOF' => ['bool'],
-'Redis::bgSave' => ['bool'],
-'Redis::bitCount' => ['int', 'key'=>'string'],
-'Redis::bitOp' => ['int', 'operation'=>'string', '...args'=>'string'],
-'Redis::bitpos' => ['int', 'key'=>'string', 'bit'=>'int', 'start='=>'int', 'end='=>'int'],
-'Redis::blPop' => ['array', 'keys'=>'string[]', 'timeout'=>'int'],
+'Redis::__construct' => ['void', 'options'=>'?array{host?:string,port?:int,connectTimeout?:float,auth?:list{string|null|false,string}|list{string},ssl?:array<string, mixed>,backoff?:array<string, mixed>}'],
```

Comment:
The `options` parameter should be optional

Reply:
👍 done